examples/jobs/httpd.yaml

```diff
@@ -0,0 +1,15 @@
+name: httpd
+containers:
+- image: centos/httpd
+services:
+- name: httpd
+  type: NodePort
+  ports:
+  - port: 8080
+jobs:
+- containers:
```

Comment:
I don't see any problems with jobs having containers? @cdrage Why do you think its confusing?